waltz-ng/client/data-types/routes.js

```diff
@@ -0,0 +1,42 @@
+const baseState = {
+    resolve: {
+        dataTypes: [
+            'DataTypesService',
+            (dataTypesService) => dataTypesService.loadDataTypes()
+        ]
+    }
+};
+
+
+const listState = {
+    url: 'data-types',
+    views: {'content@': require('./data-type-list') }
+};
+
+
+const viewByCodeState = {
+    url: 'data-types/code/{code}',
+    views: {'content@': require('./data-type-view') },
+    redirectTo: 'main.data-type.view',
```

Comment:
why do we need a redirect ?  Can't both routes point the same page/controller combo ?